modules/jupyter/jupyter_image/notebook_image/requirements.txt

```diff
@@ -0,0 +1,5 @@
+langchain==0.1.9 
```

Comment:
Why we not use the latest?

Reply:
This was the version used in the Jupyter notebook example, I didn't want to change due to the potential impact.